packages/sample/src/main.ts

```diff
@@ -20,4 +22,26 @@ program
         await led();
     });
 
+let digitalCommand = program.command("digital");
+
+digitalCommand
+    .command("write <channel> <state>")
+    .description("write digital pin")
+    .action(async (channel, state) => {
+        let channelNumber = Number(channel);
+        let stateValue = state == "high" ? DigitalState.High : DigitalState.Low;
```

Comment:
We need to document what values to pass in for `<state>`, and we should reject anything besides `high`, `low`, `1`, or `0`, and process `1` correctly.

Reply:
Added list of valid entries to help and added code to reject anything other than high, low, 0, or 1.